cmds/rm/rm.go

```diff
@@ -67,6 +74,9 @@ func rm(files []string) error {
 		}
 
 		if err := f(file); err != nil {
+			if flags.f && os.IsNotExist(err) {
```

Comment:
The force flag should also cause `rm` to ignore os.IsPermission(err) to match coreutils behavior.

Reply:
Uhm, `rm -f` coreutils doesn't silence permission errors, either:

```
--> rm -f /usr/bin/rm
rm: cannot remove '/usr/bin/rm': Permission denied
--> echo $?
1
-->    
```

Reply:
I wasn't thinking of the error message actually, but continuing onto the next file when there's an error. So looking at it again this isn't something specific to the force flag. It's a general inconsistency where its returning err instead of printing the error and continuing.

```
$ ./rm /private/tftpboot/test2 ./test1
remove /private/tftpboot/test2: permission denied
$ ls test1
test1
$ rm /private/tftpboot/test2 ./test1
override rw-r--r--  root/wheel for /private/tftpboot/test2? y
rm: /private/tftpboot/test2: Permission denied
$ ls test1
ls: test1: No such file or directory
```